docs/config.md

```diff
@@ -2,85 +2,69 @@
+| `Npgsql`             | [Npgsql](https://www.nuget.org/packages/Npgsql)                                                                                                                                                   | ≥6.0.0             | source               |
+| `SqlClient`          | [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient) and [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient)                             | *                  | source               |
+| `StackExchangeRedis` | [StackExchange.Redis](https://www.nuget.org/packages/StackExchange.Redis)  **Not supported on .NET Framework**                                                                                    | ≥2.0.405 < 3.0.0   | source & bytecode    |
+| `Wcf`                | [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **No support for server side on .NET (Core)**. For configuration see [WCF Instrumentation Configuration](wcf-config.md) | *                  | source               |
+
+### Metrics instrumentations
+
+| ID           | Instrumented library                                                                                                                                                                            | Supported versions | Instrumentation type |
+|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|----------------------|
+| `AspNet`     | ASP.NET Framework                                                                                                                                                                               | *                  | source               |
+| `AspNet`     | ASP.NET Core                                                                                                                                                                                    | *                  | source               |
+| `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | *                  | source               |
+| `NetRuntime` | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime)                                                                                   | *                  | source               |
+
+### Logs instrumentations
+
+| ID               | Instrumented library                                                                        | Supported versions | Instrumentation type   |
```

Comment:
`ID` is misleading here, should we remove `ID` column until we get the support for other logging provider and environment variables to support it.

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1311/commits/b9dc3c4ed2da4f54db716a033a3db96c90de2405